### PR TITLE
Composer: Reimplementation of whole repository to Composer package.

### PR DESCRIPTION
**Pull request by janbarasek**
Implementation of: https://github.com/gaarf/XML-string-to-PHP-array/issues/14

### CODE REVIEW

**Comment by joserick** [comment]
What happened with this change?... The Composer package will always be created?, I need it for my project [PNGMetadata](https://github.com/joserick/PNGMetadata)

**Comment by janbarasek** [comment]
Yes, package will be created automatically. You will simply use command:

composer install gaarf/xml-to-php-array

And in case of update simply “composer update”. More information about usecases is in new version of README.md, please check it.

File: README.md [hunk 1]
```diff
@@ -82,29 +113,32 @@ hereby released under a do-whatever-but-dont-sue-me license.
...
+        )
+)
+```
 
 Victory is mine! :D
 
 ---
 
 ### Contributions
+
```
Comment (gaarf) [inline README.md]: @janbarasek please add a note about your contribution here?

Comment (janbarasek) [reply README.md]: @gaarf Added: https://github.com/gaarf/XML-string-to-PHP-array/pull/15/commits/51bfc0f5deb1f1ea17252cc419d8e01789ae66b7

**Pull request merged**